VideoAPI/VideoApi.Contract/Responses/MessageResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace VideoApi.Contract.Responses
+{
+    public class MessageResponse
+    {
+        public long Id { get; set; }
```

Comment:
we do not need to expose ID

Reply:
Removed. This affects the unit tests. Please review the tests too.